src/main/scala/com/github/sbt/duplicates/DuplicatesFinderPlugin.scala

```diff
@@ -71,7 +71,7 @@ object DuplicatesFinderPlugin extends AutoPlugin {
     val classpath           = Classpath(fullClasspath.value.files ++ additionalClasspath, excludePatterns.value)
 
     Seq(classpath.classesDuplicates, classpath.resourcesDuplicates)
-      .map(_.filter(conflict => !(reportIfSameContent && conflict.conflictState == ConflictState.ContentEqual)))
+      .map(_.filter(conflict => conflict.conflictState == ConflictState.ContentDiffer || reportIfSameContent))
```

Comment:
I wasn't sure about this change. The negation of `reportIfSameContent` seemed like a bug. I could be wrong. Double check me on this.

Reply:
The behaviour stays the same (eg. report conflict if content differs or if content is equal, if it's been enabled), but yours is clearer, let's keep it :+1:

Reply:
This line is melting my brain. I tried to make a truth table manually to prove I wasn't crazy, but I screwed that up, too--so I wrote a script.

```scala
for {
  r <- Seq(true, false)
  c <- Seq("Equal", "Diff")
} {
  println(s"| $r | $c | ${!(r && c == "Equal")} | ${r || c == "Diff"} | ")
}
```

| report | ConflictState | `!(r && Equal)` | `r || Diff` |
|--------|---------------|-----------------|-------------|
| true   | Equal         | `false`         | `true`      |
| true   | Diff          | true            | true        |
| false  | Equal         | `true`          | `false️`    |
| false  | Diff          | true            | true        |

Reply:
I just guilted myself into writing a scripted test. That's the right solution to settle this. One sec.

Reply:
Okay, reverting this line causes the new scripted test to fail.
```
[info] [error] (equal/compile:checkDuplicatesTest) Detected 1 classes conflicts:
[info] [error]
[info] [error] - ConflictingFile.class: same content
[info] [error] 	 - /private/var/folders/k2/8q7nrzn13yl4byhlt8yb4mp00000gp/T/sbt_74e4bbb6/sanity/core/target/scala-2.11/classes
[info] [error] 	 - /private/var/folders/k2/8q7nrzn13yl4byhlt8yb4mp00000gp/T/sbt_74e4bbb6/sanity/equal/target/scala-2.11/classes
[info] [error] Total time: 0 s, completed Jul 19, 2020 1:34:12 PM
[error] x sanity / sanity
[error]    {line 25}  Command failed: equal/checkDuplicatesTest failed
```

I'm satisfied now that this change is correct.